### PR TITLE
Blog : Select component post

### LINKED ISSUE #3890: Blog: how to use complex types with Select component
Explain why we support primitive types only, and how to workaround to use complex types as a value.

### PR DESCRIPTION
**Pull request by David-Moreira**
Closes #3890

Can you get me some cover images for the blog post?

### CODE REVIEW

File: Documentation/Blazorise.Docs/Blazorise.Docs.csproj [hunk 1]
```diff
@@ -70,6 +70,7 @@
   </ItemGroup>
   <ItemGroup>
     <Folder Include="Pages\Blog\2022-06-09_ValidationWithDataAnnotations\Code\" />
+    <Folder Include="Pages\Blog\UsingTheSelectComponent\Code\" />
```
Comment (stsrki) [inline Documentation/Blazorise.Docs/Blazorise.Docs.csproj:73]: Both of these `\Code\` folders should be removed.

File: Documentation/Blazorise.Docs/Pages/Blog/UsingTheSelectComponent/Index.razor [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+<BlogPageSubtitle>
+    Complex Types
+</BlogPageSubtitle>
+
+<BlogPageParagraph>
+    Blazorise doesn't currently support binding a complex type directly, this is because the select element has no way of uniquely identifying an item of a complex type, so we have to uniquely identify the item ourselves.
+    Let's take a look at how a dropdown-list with four employees would look, now with actual complex types.
+</BlogPageParagraph>
+
+<UsingTheSelectComponentComplexTypeExample />
```
Comment (stsrki) [inline Documentation/Blazorise.Docs/Pages/Blog/UsingTheSelectComponent/Index.razor:60]: Rename `UsingTheSelectComponentComplexTypeExample` into `SelectComponentWithComplexTypeExample`. Same for other examples.

File: Documentation/Blazorise.Docs/Pages/Blog/UsingTheSelectComponent/Examples/UsingTheSelectComponentComplexTypeExample.razor [hunk 3]
```diff
@@ -0,0 +1,42 @@
+﻿@namespace Blazorise.Docs.Pages.Blog.UsingTheSelectComponent.Examples
+
+<Row>
+    <Column>
+        <Select TValue="int"
```
Comment (stsrki) [inline Documentation/Blazorise.Docs/Pages/Blog/UsingTheSelectComponent/Examples/UsingTheSelectComponentComplexTypeExample.razor:5]: When you have any input component tend to wrap it with a `Field`. That is always recommended.

**Comment by stsrki** [comment]
Also, add your image for an author :)

**Comment by David-Moreira** [comment]
Reminder : Briefly explain primitive (known as value types) vs complex types (reference types)

**Pull request merged**